src/requirements/checkers/checkers-common.ts

```diff
@@ -43,3 +43,13 @@ export const courseIsFWS = (course: Course): boolean => (
   course.titleLong.includes('FWS:')
   || (course.catalogSatisfiesReq?.includes('First-Year Writing Seminar') ?? false)
 );
+
+export const includesWithSingleRequirement = (...includes: readonly string[]) => (course: Course): boolean => (
+  courseMatchesCodeOptions(course, includes)
+);
+
+export const includesWithSubRequirements = (...includes: readonly string[][]): readonly ((course: Course) => boolean)[] => (
+  includes.map(
+    subRequirementInclude => (course: Course) => courseMatchesCodeOptions(course, subRequirementInclude)
+  )
+);
```

Comment:
All functions in this file will be referenced to a lot. For the sake of clarity to the requirement adders, can we get specifications for these two new functions too?